easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2023.12.1+402-gfbf-2023b-Java-11-R-4.3.3.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'CMakeNinja'
+
+name = 'RStudio-Server'
+version = "2023.12.1+402"
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+local_git_rev = '4da58325ffcff29d157d9264087d4b1ab27f7204'
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port=8787
+
+If you need a database config one can be created with:
+  MYTMP=`mktemp -d` && echo -e "provider=sqlite\\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf"
+and then used with:
+  rserver ... --database-config-file="${MYTMP}/db.conf"
+"""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+source_urls = ['https://github.com/rstudio/rstudio/archive']
+sources = ['%(version)s.tar.gz']
+
+checksums = []
+builddependencies = [
+    ('ant', '1.10.14', '-Java-%(javaver)s', SYSTEM),
+    ('CMake', '3.27.6'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '2.0.3'),
+    ('nodejs', '20.9.0'),
+]
+
+dependencies = [
+    ('Boost', '1.83.0'),
+    ('Java', '11', '', SYSTEM),
+    ('R', '4.3.3'),
+    ('SOCI', '4.0.3'),
+    ('yaml-cpp', '0.8.0'),
+]
+
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev')
+]
+
+# Required until https://github.com/easybuilders/easybuild-framework/pull/4035
+local_start_dir = "%(builddir)s/rstudio-" + version.replace("+", "-")
```

Comment:
That PR is already merged and released in 4.7.1, so I guess it can be removed

Reply:
Done